bin/add_host_to_ssh_known_hosts

```diff
@@ -7,4 +7,4 @@ URL="${1:?You need to provide an URL as first parameter}"
 [[ $URL =~ ^(https?://|git@)?([^/:]+) ]] && HOST=${BASH_REMATCH[2]}
 
 echo "Adding ${HOST} to '~/.ssh/known_hosts'..."
-for ip in $(dig @8.8.8.8 "${HOST}" +short); do ssh-keyscan "${HOST}",$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts || true
+for ip in $(dig @8.8.8.8 "${HOST}" +short); do ssh-keyscan "${HOST}","$ip"; ssh-keyscan "$ip"; done 2>/dev/null >> ~/.ssh/known_hosts || true
```

Comment:
Hmm – assuming a compatible `dig` it shouldn't be possible for the IP address to have spaces 🤔 

That said, I can see doing this for style reasons, but it's _technically_ not necessary...

Reply:
Yep I agree. But I'd still do it for style reasons … and for making shellcheck happy, especially if we plan to make it run regularly via CI at some point